### PR TITLE
Rename irb120

### PR DESCRIPTION
**Pull request by Jmeyer1292**
Do not merge.

This PR changes the mainline package (`abb_irb120_support`) to comply with the new name rules.

Do we want to update both moveit-configs and the gazebo package as well? Or shall we drop them?

### CODE REVIEW

**Comment by gavanderhoorn** [comment]
> Do we want to update both moveit-configs and the gazebo package as well? Or shall we drop them?

my vote would be for updating them.

**Comment by gavanderhoorn** [comment]
Travis failed because the `roslaunch_test.xml` file is not including the updated filenames for the IRB 120.

File: abb_irb120_moveit_config/config/joint_limits.yaml [hunk 1]
```diff
@@ -1,31 +1,34 @@
...
+# Specific joint properties can be changed with the keys [max_position, min_position, max_velocity, max_acceleration]
+# Joint limits can be turned off with [has_velocity_limits, has_acceleration_limits]
 joint_limits:
   joint_1:
     has_velocity_limits: true
     max_velocity: 4.36332
-    has_acceleration_limits: true
-    max_acceleration: 0.87266
+    has_acceleration_limits: false
+    max_acceleration: 0
```
Comment (Levi-Armstrong) [inline abb_irb120_moveit_config/config/joint_limits.yaml]: Do you know why this changed for the acceleration limits; I assume you just re-ran MoveIt Setup Assistant:

File: abb_irb120_moveit_config/package.xml [hunk 2]
```diff
@@ -1,39 +1,32 @@
...
+  <author email="culletom@tcd.ie">Mark Culleton (Trinity College Dublin)</author>
   <maintainer email="culletom@tcd.ie">Mark Culleton (Trinity College Dublin)</maintainer>
 
   <license>BSD</license>
 
   <url type="website">http://moveit.ros.org/</url>
-  <url type="bugtracker">https://github.com/ros-industrial/abb_experimental/issues</url>
-  <url type="repository">https://github.com/ros-industrial/abb_experimental</url>
+  <url type="bugtracker">https://github.com/ros-planning/moveit/issues</url>
+  <url type="repository">https://github.com/ros-planning/moveit</url>
```
Comment (Levi-Armstrong) [inline abb_irb120_moveit_config/package.xml]: Should this be changed back to point to the abb_experimental?

File: abb_irb120t_moveit_config/config/joint_limits.yaml [hunk 3]
```diff
@@ -1,31 +1,34 @@
...
+# Specific joint properties can be changed with the keys [max_position, min_position, max_velocity, max_acceleration]
+# Joint limits can be turned off with [has_velocity_limits, has_acceleration_limits]
 joint_limits:
   joint_1:
     has_velocity_limits: true
     max_velocity: 4.36332
-    has_acceleration_limits: true
-    max_acceleration: 0.87266
+    has_acceleration_limits: false
+    max_acceleration: 0
```
Comment (Levi-Armstrong) [inline abb_irb120t_moveit_config/config/joint_limits.yaml]: Acceleration limits same as above.

File: abb_irb120t_moveit_config/package.xml [hunk 4]
```diff
@@ -1,39 +1,32 @@
...
+  <author email="culletom@tcd.ie">Mark Culleton (Trinity College Dublin)</author>
   <maintainer email="culletom@tcd.ie">Mark Culleton (Trinity College Dublin)</maintainer>
 
   <license>BSD</license>
 
   <url type="website">http://moveit.ros.org/</url>
-  <url type="bugtracker">https://github.com/ros-industrial/abb_experimental/issues</url>
-  <url type="repository">https://github.com/ros-industrial/abb_experimental</url>
+  <url type="bugtracker">https://github.com/ros-planning/moveit/issues</url>
+  <url type="repository">https://github.com/ros-planning/moveit</url>
```
Comment (Levi-Armstrong) [inline abb_irb120t_moveit_config/package.xml]: Same as above.

Comment (gavanderhoorn) [reply abb_irb120t_moveit_config/package.xml]: @Jmeyer1292: easiest would probably to revert the changes to `package.xml`. The new MSA doesn't add anything beyond what was already in the manifest.

File: abb_irb120_moveit_config/config/joint_limits.yaml (hunk 1 above)
Comment (gavanderhoorn) [reply abb_irb120_moveit_config/config/joint_limits.yaml]: This could be reverted as well.

Although those limits are made up, just as those for the 120t.

**Comment by Jmeyer1292** [comment]
I reverted the `package.xml` and `joint_limits.yaml` files in the moveit-config packages.

**Comment by Jmeyer1292** [comment]
The moveit packages appear to work as expected. There appears to be a problem running the gazebo package w/ moveit. Some controller action name is missing. I need to investigate if its something I caused.

**Comment by Jmeyer1292** [comment]
The same issue with the gazebo/moveit controller connections not working is present in `kinect-devel`. I think this one is ready to go.

**Comment by Jmeyer1292** [comment]
Are there other concerns? @Levi-Armstrong please review.

**Comment by Levi-Armstrong** [comment]
:+1:

**Pull request merged**